src/tests/components/SearchForm.test.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import SearchForm from "../../components/SearchForm";
+
+describe("SearchForm", () => {
+  const validProps = {
+    searchText: "Manchester",
+    setSearchText: () => {},
+    onSubmit: () => {},
+  };
+
+  it("renders correctly", () => {
+    const { asFragment } = render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.onSubmit}
+      />
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("checks in inner button text", () => {
+    render(
+      <SearchForm
+        searchText={validProps.searchText}
+        setSearchText={validProps.setSearchText}
+        onSubmit={validProps.onSubmit}
+      />
+    );
+    const buttons = screen.getAllByRole("button");
+    expect(buttons[0]).toHaveTextContent("Search");
```

Comment:
Its common convention to leave a clear line before the assertions start in each test to help with readability, should add one in here